o2a/mappers/action_mapper.py

```diff
@@ -39,20 +44,21 @@ def __init__(self, oozie_node: Element, name: str, dag_name: str, props: Propert
 
     def on_parse_node(self, workflow: Workflow):
         super().on_parse_node(workflow)
-        self._parse_config()
+        self._parse_config(workflow)
+
+    def _parse_config(self, workflow: Workflow):
+        job_xml_nodes = find_nodes_by_tag(self.oozie_node, TAG_JOB_XML)
+        new_properties = extract_properties_from_job_xml_nodes(
```

Comment:
`new_properties` -> `job_xml_properties` ?

Reply:
I updated the PR.